modules/virt-configuring-ip-for-vm.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-configuring-ip-for-vm_{context}"]
+= Configuring an IP address for a virtual machine
+
+You can use cloud-init to configure an IP address for a virtual machine. The IP address can be provisioned to be dynamic or static. 
+
+You can add or edit the cloud-init configuration of an existing virtual machine configuration. If you edit a running virtual machine, the virtual machine must be rebooted for the changes to take effect.
+
+.Prerequisites
+
+* DNS is configured via DHCP on the cluster.
+
+* A Linux bridge network attachment definition.
+
+* The virtual machine is connected to the bridge network. 
```

Comment:
Say it is connected to a secondary network as described in https://docs.openshift.com/container-platform/4.6/virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.html